### PR TITLE
OSDOCS#4554: Add an overview of OpenShift with a diagram

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.8+

Issue: https://issues.redhat.com/browse/OSDOCS-4554

Link to docs preview: https://52772--docspreview.netlify.app/openshift-enterprise/latest/getting_started/openshift-overview.html#understanding-openshift_openshift-overview

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/understanding-openshift.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+:_content-type: CONCEPT
+[id="understanding-openshift_{context}"]
+= Understanding {product-title}
+
+{product-title} is a Kubernetes environment for managing the lifecycle of container-based applications and their dependencies on various computing platforms such as bare metal, virtualized, on-premise, and in the cloud. {product-title} deploys, configures and manages containers. {product-title} offers usability, stability, and customization of its components.
+
+{product-title} utilises a number of computing resources, known as nodes. The nodes has a lightweight, secure operating system based on {op-system-base-full}, known as {op-system-first}.
+
+After a node is booted and configured, it obtains a container runtime, such as CRI-O or Docker, for managing and running the images of container workloads scheduled to it. Kubernetes agent, or kubelet, schedules container workloads on the node. Kubelet is responsible for registering the node with the cluster and receiving the details of container workloads.
```
Comment (pjwebster) [inline modules/understanding-openshift.adoc:13]: Review at the request of sdudhgao@redhat.com

`
[The] Kubernetes agent, or kubelet, schedules container workloads on the node. [The] kubelet is responsible for registering the node with the cluster and receiving the details of container workloads.
`

Comment (xenolinux) [reply modules/understanding-openshift.adoc]: Addressed

File: modules/understanding-openshift.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+{product-title} configures and manages the networking, load balancing and routing of the cluster. {product-title} adds cluster services for monitoring the cluster health and performance, logging, and for managing upgrades.
+
+The container image registry and OperatorHub provide Red Hat certified products and community built softwares for providing various application services within the cluster. These applications and services manage the applications deployed in the cluster, databases, frontends and user interfaces, application runtimes and business automation, and developer services for development and testing of container applications.
+
+You can manage applications within the cluster manually by configuring deployments of containers running from pre-built images. You can manage applications through resources known as Operators. You can build custom images from pre-build images and source code. You can store these custom images locally in an internal, private or public registry.
```
Comment (snarayan-redhat) [inline modules/understanding-openshift.adoc:19]:
```suggestion
You can manage applications within the cluster either manually by configuring deployments of containers running from pre-built images or through resources known as Operators. You can build custom images from pre-build images and source code, and store these custom images locally in an internal, private or public registry.
```

**Review by snarayan-redhat — COMMENTED** [review]
Great job! 
Have provided few suggestions.

File: modules/understanding-openshift.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+After a node is booted and configured, it obtains a container runtime, such as CRI-O or Docker, for managing and running the images of container workloads scheduled to it. The Kubernetes agent, or kubelet, schedules container workloads on the node. The kubelet is responsible for registering the node with the cluster and receiving the details of container workloads.
+
+{product-title} configures and manages the networking, load balancing and routing of the cluster. {product-title} adds cluster services for monitoring the cluster health and performance, logging, and for managing upgrades.
+
+The container image registry and OperatorHub provide Red Hat certified products and community built softwares for providing various application services within the cluster. These applications and services manage the applications deployed in the cluster, databases, frontends and user interfaces, application runtimes and business automation, and developer services for development and testing of container applications.
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/understanding-openshift.adoc:17]: I think it is valid as is community built

File: modules/understanding-openshift.adoc (hunk 2 above)
Comment (xenolinux) [reply modules/understanding-openshift.adoc]: ack

File: modules/understanding-openshift.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/understanding-openshift.adoc:17]: ack

File: modules/understanding-openshift.adoc [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+:_content-type: CONCEPT
+[id="understanding-openshift_{context}"]
+= Understanding {product-title}
+
+{product-title} is a Kubernetes environment for managing the lifecycle of container-based applications and their dependencies on various computing platforms such as bare metal, virtualized, on-premise, and in the cloud. {product-title} deploys, configures and manages containers. {product-title} offers usability, stability, and customization of its components.
+
+{product-title} utilises a number of computing resources, known as nodes. The nodes has a lightweight, secure operating system based on {op-system-base-full}, known as {op-system-first}.
+
+After a node is booted and configured, it obtains a container runtime, such as CRI-O or Docker, for managing and running the images of container workloads scheduled to it. The Kubernetes agent, or kubelet, schedules container workloads on the node. The kubelet is responsible for registering the node with the cluster and receiving the details of container workloads.
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/understanding-openshift.adoc]: ack

File: modules/understanding-openshift.adoc [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+//
+// * getting-started/openshift-overview.adoc
+
+:_content-type: CONCEPT
+[id="understanding-openshift_{context}"]
+= Understanding {product-title}
+
+{product-title} is a Kubernetes environment for managing the lifecycle of container-based applications and their dependencies on various computing platforms such as bare metal, virtualized, on-premise, and in the cloud. {product-title} deploys, configures and manages containers. {product-title} offers usability, stability, and customization of its components.
+
+{product-title} utilises a number of computing resources, known as nodes. The nodes has a lightweight, secure operating system based on {op-system-base-full}, known as {op-system-first}.
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/understanding-openshift.adoc]: ack

File: modules/understanding-openshift.adoc [hunk 6]
```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-overview.adoc
+
+:_content-type: CONCEPT
+[id="understanding-openshift_{context}"]
+= Understanding {product-title}
+
+{product-title} is a Kubernetes environment for managing the lifecycle of container-based applications and their dependencies on various computing platforms such as bare metal, virtualized, on-premise, and in the cloud. {product-title} deploys, configures and manages containers. {product-title} offers usability, stability, and customization of its components.
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/understanding-openshift.adoc]: ack

[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/understanding-openshift.adoc]: ack

**Review by pjwebster — APPROVED** [review]
Looks good!

**Comment by mburke5678** [comment]
@xenolinux Do you want to take on the manual cherrypicks for 4.8 and 4.9?

**Pull request merged**